snuba/datasets/cdc/groupedmessage_processor.py

```diff
@@ -1,54 +1,76 @@
+from datetime import datetime
+from typing import Any, Mapping, Optional
+
+from dataclasses import dataclass
 from dateutil.parser import parse as dateutil_parse
-from snuba.datasets.cdc.cdcprocessors import CdcProcessor
+from snuba.datasets.cdc.cdcprocessors import CdcProcessor, CDCMessageRow
 
 
-class GroupedMessageProcessor(CdcProcessor):
+@dataclass(frozen=True)
+class GroupedMessageRow(CDCMessageRow):
```

Comment:
Maybe we will be able to move the clickhouse schema here and make it mostly automatically generated.